frontend/src/metabase/query_builder/components/view/ExecutionTime/ExecutionTime.unit.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import { ExecutionTime } from "./ExecutionTime";
+
+describe("ExecutionTime", () => {
+  it("renders nothing when no time is provided (metabase#45730)", () => {
+    const { container } = render(<ExecutionTime />);
+
+    expect(container).toBeEmptyDOMElement();
+  });
+
+  it("renders formatted time when time is provided", () => {
+    render(<ExecutionTime time={100} />);
```

Comment:
Please add a test case where `time={0}`

Reply:
Ok, but it makes sense to add it here, right?
https://github.com/metabase/metabase/blob/master/frontend/src/metabase/query_builder/components/view/ExecutionTime/utils.unit.spec.ts

Reply:
Here because `if (!time)` check lives in the component.

Reply:
Good catch.
I guess the question is do we want to display "0 ms" or do we want to not render the component. The solution depends on that answer.

Reply:
"0 ms" please :pray:

Reply:
https://github.com/metabase/metabase/pull/45749/commits/88506425652f9b343f9ad57e97f4212c66769065